.gitignore

```diff
@@ -19,3 +19,4 @@
 
 # Go workspace file
 go.work
+.idea
```

Comment:
for better experience, add this to your global ignore file. But I will update local too. For now please remove.